[2022-02-09 08:58:59,086] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-06-01T23:00:00+00:00 [queued]>
[2022-02-09 08:58:59,100] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-06-01T23:00:00+00:00 [queued]>
[2022-02-09 08:58:59,100] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:58:59,100] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:58:59,101] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:58:59,118] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-06-01 23:00:00+00:00
[2022-02-09 08:58:59,125] {standard_task_runner.py:52} INFO - Started process 145 to run task
[2022-02-09 08:58:59,132] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-06-01T23:00:00+00:00', '--job-id', '455', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpmbn3vsl9', '--error-file', '/tmp/tmpcybl0r7n']
[2022-02-09 08:58:59,134] {standard_task_runner.py:77} INFO - Job 455: Subtask bigquery_external_table_task
[2022-02-09 08:58:59,226] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-06-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:58:59,295] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:58:59,326] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-06-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-01T23:00:00+00:00
[2022-02-09 08:58:59,328] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 08:59:00,014] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20200601T230000, start_date=20220209T085859, end_date=20220209T085900
[2022-02-09 08:59:00,071] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:00,122] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:14:06,134] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-06-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:06,156] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-06-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:06,156] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:06,156] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:14:06,157] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:06,180] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-06-01 23:00:00+00:00
[2022-02-09 09:14:06,188] {standard_task_runner.py:52} INFO - Started process 210 to run task
[2022-02-09 09:14:06,203] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-06-01T23:00:00+00:00', '--job-id', '544', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpr569jges', '--error-file', '/tmp/tmpssvgrlys']
[2022-02-09 09:14:06,205] {standard_task_runner.py:77} INFO - Job 544: Subtask bigquery_external_table_task
[2022-02-09 09:14:06,363] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-06-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:14:06,433] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:06,458] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-06-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-01T23:00:00+00:00
[2022-02-09 09:14:06,460] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:14:08,035] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20200601T230000, start_date=20220209T091406, end_date=20220209T091408
[2022-02-09 09:14:08,066] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:08,105] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
